runtime/polkadot/src/lib.rs

```diff
@@ -938,6 +967,13 @@ construct_runtime! {
 		// Old spot for the purchase pallet. Can be replaced later by a new pallet.
 		DummyPurchase: dummy::<Instance4>::{Module, Call, Event<T>},
 
+		// Parachains runtime modules
+		Configuration: configuration::{Module, Call, Storage},
```

Comment:
Why did you added all this stuff? This makes this pr "unmergable". 

This will require a lot of testing and are these pallets already ready?

They also change the call encoding in an invasive way.

Reply:
I added these because of https://github.com/paritytech/polkadot/pull/1630/#discussion_r487049798 and https://github.com/paritytech/polkadot/pull/1630/#discussion_r488728339. If that's the wrong action, it's easy to revert.

I mentioned earlier: I don't have any intuitive understanding of what the consequences of that change will be. The code compiles, which is clearly a necessary but sufficient step. I don't know what the rest of the steps are.

Reply:
I also don't believe we should yet alter the Polkadot or Kusama runtime module sets, if possible. It would make most sense to me to provide a dummy implementation of the `v1::ParachainHost` in each of those runtimes, but other than that not to alter them.

Reply:
Just merge master to get these dummy implementations of the runtime apis.

Reply:
Done in df7a758... but something's up with CI, apparently; the checks don't appear to be running.